nat-lab/tests/test_lana.py

```diff
@@ -283,10 +306,15 @@ async def run_default_scenario(
         client_beta.wait_for_state_on_any_derp([telio.State.Connected]),
         client_gamma.wait_for_state_on_any_derp([telio.State.Connected]),
     )
+    # Note: GAMMA is symmetric, so it will not connect to ALPHA or BETA in diret mode
     await asyncio.gather(
-        client_alpha.wait_for_state_peer(beta.public_key, [telio.State.Connected]),
+        client_alpha.wait_for_state_peer(
+            beta.public_key, [telio.State.Connected], [PathType.Direct], timeout=60.0
```

Comment:
nit: timeout default is 60 already for direct connection types

Reply:
Fixed